config/config_gaea.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+# Compiler/MPI combination
+export HPC_COMPILER="intel/18.0.6.288"
+export HPC_MPI="impi/2018.0.4"
```

Comment:
This line needs to be replaced with
```
export HPC_MPI="cray-mpich/7.7.11"
```
Loading impi probably has no effect, due to the Cray compiler wrappers being used as CC,FC,CXX (serial, MPI) below, but we need to make sure we load the correct cray mpich.

Reply:
Yeah. Definitely need to load the cray-mpich module correctly

Reply:
done in 1dc96f0